### DISCUSSION TITLE
👽 Apify's Instagram Scraper and Instagram Reel Scraper

### DISCUSSION THREAD

**Discussion by chriskyfung** [chriskyfung/AutoFetcher-IG-Stories-to-GDrive#135] [Literature]
Hi everyone,

I wanted to start a discussion about the tools available on Apify for scraping Instagram data, specifically the **Instagram Scraper** and **Instagram Reel Scraper**. These tools are sold for extracting data from Instagram, including posts, profiles, hashtags, or reels.

**Scalability**: The Instagram Reel Scraper can return up to 10,000 results, making it suitable for large-scale data collection.

Check out the [Instagram Scraper](https://apify.com/apify/instagram-scraper) and [Instagram Reel Scraper](https://apify.com/apify/instagram-reel-scraper) on Apify.

Let’s share any experiences you might have and discuss tips and tricks for using these tools effectively!

**Comment by chriskyfung** [comment]
## Some useful references:

### 📰 [How to scrape Instagram data: profiles, posts, comments](https://blog.apify.com/scrape-instagram-posts-comments-and-more-21d05506aeb3/)

Here are the key points from the Apify's blog post:

- **Instagram Scraper**: An unofficial tool developed by Apify to extract data from Instagram profiles, posts, comments, and media without needing deep technical knowledge.
- **Limitations of Official APIs**: Instagram's Basic Display API and Graph API have significant limitations, such as not being able to scrape regular user data or requiring technical setup.
- **Ease of Use**: The Instagram Scraper simplifies the process, allowing users to scrape data by just inputting profile URLs and setting parameters.
- **Ethical Considerations**: Emphasizes the importance of ethical scraping, avoiding personal data extraction, and adhering to legal guidelines.

Here are some examples of data you can scrape using the Instagram Scraper without login:

- **Profile Information**: Username, bio, profile picture, follower count, following count, and profile URL.
- **Posts**: Captions, media type (photo/video), media URL, timestamp, and number of likes.
- **Comments**: Text of the comments, usernames of commenters, and timestamps.
- **Hashtags**: List of hashtags used in posts and their frequency.

### [How to scrape Instagram posts for free](https://blog.apify.com/how-to-scrape-instagram-posts-in-5-easy-steps/)

Here are the key points from the tutorial on extracting Instagram post data using Apify:

- **Getting Started**: Sign up for a free Apify account to use the tool. No credit card is required.
- **Steps to Use**: 
  1. **Select the Tool**: Choose Instagram Post Scraper from the Apify Store.
  2. **Input Data**: Enter the Instagram username(s) you want to scrape.
  3. **Set Limits**: Specify the maximum number of posts to extract.
  4. **Run and Export**: Start the scraper and download the data once the process is complete.

**Reply by coreclaw888** [reply]
Wow, that's great! Could you try using Coreclaw instead of Apify? It might surprise you.

https://www.coreclaw.com/coreclaw/google-maps-scraper?utm_source=google&utm_medium=referral&utm_campaign=email&utm_term=&utm_id=email
